documentation/asciidoc/computers/camera/synchronous_cameras.adoc

```diff
@@ -0,0 +1,83 @@
+== Synchronous Captures
+=== IMX477
+==== Preparation
+
+Connect a potential divider of 2x 1.5KOhm Resistors to 3v3 and ground, creating 1.65V. This can be connected to either Pi.
```

Comment:
This definitely feels like this page is missing an introductory section, before diving straight into details about connecting resistors? :thinking: